support/modules/single-file

```diff
@@ -53,8 +64,7 @@ case "$STATE" in
         dest_dir=$(cat "${dest_dir_file}")
         test -z "$dest_dir" -o -z "$filename" && \
             echo "Fatal error: dest_dir or filename are undefined." && exit 1
```

Comment:
I know it's not on you, but could you add an extra commit with echo to `stderr` here? :pray:

Reply:
I will!